.golangci.yaml

```diff
@@ -7,12 +7,9 @@ run:
 linters:
   disable-all: true
   enable:
-    - deadcode
```

Comment:
@rohan47 it looks like the changes in this file are enough to pass the CI, can you please revert the changes in the golangcilint.yml file in action?

Reply:
done